package.json

```diff
@@ -170,6 +171,7 @@
     "less": "2.7.1",
     "less-loader": "4.1.0",
     "lodash": "npm:@elastic/lodash@3.10.1-kibana1",
+    "lodash.clonedeep": "^4.5.0",
```

Comment:
Isn't cloneDeep a function of lodash, and we already have lodash as a dependency. Why are we adding this?